hr_employee_lastnames/__manifest__.py

```diff
@@ -3,7 +3,7 @@
     "name": "HR Employee First Name and Two Last Names",
     "version": "16.0.1.0.1",
     "author": "Vauxoo, Odoo Community Association (OCA)",
-    "maintainer": "Vauxoo",
+    "maintainers": ["Vauxoo", "luisg123v"],
```

Comment:
`vauxoo` is not actually an expected GitHub username, please remove it.

Reply:
Done.